test/views/contactNumbers/ContactNumbersChangeSuccessViewSpec.scala

```diff
@@ -68,7 +68,7 @@ class ContactNumbersChangeSuccessViewSpec extends ViewBaseSpec {
           }
 
           "have the correct first paragraph" in {
-            elementText(Selectors.paragraphOne) shouldBe "We will send you an email within 2 working days with an update," +
+            elementText(Selectors.paragraphOne) shouldBe "We‘ll send you an email within 2 working days with an update," +
```

Comment:
I know using this type of apostrophe prevents us from having to escape, but unfortunately it also renders as that in the view instead of the flying dog tooth which can be seen in the prototype. https://vatvc-prototype.herokuapp.com/build/chocs/confirmation-tel